Gopkg.lock

```diff
@@ -49,6 +49,14 @@
   revision = "aebefd9fcb99f22cd691ef778a12ed68f0e6a1ab"
   version = "v1.3.4"
 
+[[projects]]
+  digest = "1:b98ee2c3f1469ab3b494859d3a9c9e595ec2c63660dea130a5154cfcd427b608"
+  name = "github.com/DataDog/viper"
+  packages = ["."]
+  pruneopts = ""
+  revision = "6d33b5a963d922d182c91e8a1c88d81fd150cfd4"
+  version = "v1.3.1"
```

Comment:
anything we can expect to see change/break between versions `1.0.2` and `1.3.1` of viper?

Reply:
The changelog is mainly new feature we don't use or bugfix. We should be good.